packages/Checkboxes/src/Checkbox.tsx

```diff
@@ -51,7 +64,9 @@ const Checkbox = ({ checked, value, disabled, required, label, show, name, id, o
         {...attrs}
       />
       <div onClick={handleCheckboxChange} className={labelClassName} data-testid="form-field-wrapper">
-        <Label defaultId={inputId} label={label} />
+        <Tooltip tooltipContent={tooltip} position="top">
```

Comment:
As discussed on call, update `tooltip` to be an object then this would become `<Tooltip position="top" {...tooltip}>`

we can change this everywhere